CDP4JsonSerializer/CDP4JsonSerializer.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;net471;net472;net48;netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
     <Company>RHEA System S.A.</Company>
     <Title>CDP4JsonSerializer Community Edition</Title>
-    <VersionPrefix>7.4.0</VersionPrefix>
+    <VersionPrefix>7.5.0</VersionPrefix>

```

Comment:
i have the impression that nothing has changed in the CDP4JsonSerializerproject, so why bump the version? if nothing has changed, please revert